### ISSUE TITLE
Add Hook to the list of available CLI commands

### ISSUE THREAD

**Issue by laurwang** [awslabs/aws-lambda-fanout#16]
Thank you for making this very useful feature.  Could include all of the available commands--hook, unhook, pause, resume--in "The CLI" section?  

TL;DR: When I was setting it up, I scratched my head for a while about why nothing was happening and, after some investigation, it turned out that, although I had registered a stream to stream mapping, I had not hooked the source stream up to the fanout lambda.  The reason I hadn't was a failure to scroll to the very end of the readme, where the hook command was described for the first time.  It would have helped me to have the need to hook the stream up mentioned somewhere else nearer the top of the readme, e.g., if it were included in the list of "Available commands" in "The CLI" section.  Thanks again for the feature.

**Comment by julienlepine** [comment]
Sorry I did not see your issue in my stream and left it hanging for so long. Yess I'll have the documentation fixed. Thanks
